o/opentelemetry-go/opentelemetry-go_ubi_8.5.sh

```diff
@@ -17,10 +17,11 @@
 #             contact "Maintainer" of this script.
 #
 # ----------------------------------------------------------------------------
-#Run the script:./opentelemetry-go_ubi_8.5.sh v0.13.0(version_to_test)
+#Run the script:./opentelemetry-go_ubi_8.5.sh v1.11.1(version_to_test)
+#For package version 0.*.* use go version(1.17.4)
 PACKAGE_NAME=opentelemetry-go
-PACKAGE_VERSION=${1:-v0.13.0}
-GO_VERSION=1.17.4
+PACKAGE_VERSION=${1:-v1.11.1}
+GO_VERSION=1.18.5
```

Comment:
@Pratikt2312 
Can you make GO_VERSION as configurable as :
`GO_VERSION=${GO_VERSION:-1.18.5}`

CC: @seth-priya, @nikhil-kalbande

Reply:
Ok I will do.

Reply:
Updated.  @nikhil-kalbande can u please review it once.